src/twisted/python/test/test_release.py

```diff
@@ -379,43 +390,45 @@ def test_build(self):
 
         inputPath = FilePath(self.mktemp()).child(packageName)
         inputPath.makedirs()
-        inputPath.child("__init__.py").setContent(
+        inputPathContent = (
             "def foo():\n"
             "    '%s'\n"
             "def _bar():\n"
             "    '%s'" % (docstring, privateDocstring))
-
+        setStrContent(inputPath.child("__init__.py"), inputPathContent)
```

Comment:
This binds another name in a very long function, which means one more thing for a reader to scan up and down to see if it's used.

My preference would be to do something like

```python
        setStrContent(
              inputPath.child("__init__.py"),
              "def foo():\n"		              "def foo():\n"
              "    '%s'\n"		              "    '%s'\n"
              "def _bar():\n"		              "def _bar():\n"
              "    '%s'" % (docstring, privateDocstring)
        )
```
to keep it an expression and avoid the subsequently unused `inputPathContent` name.